packages/documentation/docs/concepts/asset.md

```diff
@@ -0,0 +1,8 @@
+# Asset
```

Comment:
just took from the old glossary, still needs more additions. issue created: https://github.com/interledger/rafiki/issues/1430